OpenRA.Mods.Common/Scripting/Global/ActorGlobal.cs

```diff
@@ -64,14 +64,53 @@ public Actor Create(string type, bool addToWorld, LuaTable initTable)
 			return a;
 		}
 
-		[Desc("Returns the build time (in ticks) of the requested unit type.")]
-		public int BuildTime(string type)
+		[Desc("Returns the build time (in ticks) of the requested unit type."
+			+ "An optional second value can be used to exactly specify the producing queue type.")]
```

Comment:
When you split lines with a `+`, you need to leave a space either at the end of the preceding or the beginning of the following string. When you use a `,` to split strings, you don't.